Tasks/RunDistributedTests/task.json

```diff
@@ -160,6 +205,27 @@
          "endpoint": "/$(system.teamProject)/_apis/vslabs/environments", 
          "selector": "jsonpath:$.value[*].name", 
          "authKey": "tfs:DevTestLabs" 
+     },
+     {
+         "target": "testPlan",
+         "endpoint":"/$(system.teamProject)/_apis/test/plans",
+         "selector":"jsonpath:$.value[*].name",
+         "keySelector":"jsonpath:$.value[*].id",
+         "authKey": "tfs:DevTestLabs"
```

Comment:
The refresh button is tied to this auth key. we can add our own key and get rid of this. I'll take this as a bug fix.